bitnami/elasticsearch/templates/configmap.yaml

```
@@ -0,0 +1,23 @@
+{{- if or .Values.config .Values.extraConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
```

In order to allow the use of `namespaceOverride`: 
```suggestion
  namespace: {{ include "common.names.namespace" . | quote }}
```

NOTE: In some kubernetes objects namespace is missing